tests/security/secure_boot.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test the signed/unsigned module and kernel if can be loaded when secure-boot is enabled
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+    select_console 'root-console';
+
+    zypper_call('in pesign-obs-integration');
+
+    # Check if secure boot is enabled
+    validate_script_output('dmesg | grep -i secure', sub { m/Secure boot mode enabled/ });
+    validate_script_output('hexdump -C /proc/device-tree/ibm,secure-boot', sub { m/\d+\s+\d+\s+\d+\s+\d+\s+\d+/ });
+
+    # generate key
+    my $work_dir = "/root/certs";
+    my $cert_cfg = "$work_dir/secure_boot_cert.conf";
+    my $priv_key = "$work_dir/key.pri";
+    my $out_key = "$work_dir/key.der";
+    my $config_file = 'openssl/gencert_conf/secure_boot_cert.conf';
+
+    assert_script_run("mkdir -p $work_dir");
+    assert_script_run("wget --quiet " . data_url($config_file) . " -O $cert_cfg");
+    assert_script_run("openssl req -new -nodes -utf8 -sha256 -days 36500 -batch -x509 -config $cert_cfg -outform DER -out $out_key -keyout $priv_key");
```

Comment:
@Amrysliu 

opensslca.pm may do the same thing, can you please double confirm? 
but not a big deal

Reply:
There are some differences between them, so I prefer to keep it.